dias/utils/tracker.py

```diff
@@ -262,14 +262,17 @@ def insert_files(self, files):
 
         file_type_rows = self._get_filetypes()
 
-        self.logger.info("Adding {0} to database".format(len(files)))
+        self.logger.debug("Adding {0} DIAS input files to database".format(len(files)))
 
         # remove already logged files from list
         files_in_table = [f.filepath for f in File.select()]
 
         files = [f for f in files if f not in files_in_table]
 
         for f in files:
+            if os.path.isfile(f + ".lock"):
```

Comment:
The tracker imports and uses both `os` and `pathlib` for working with file paths. Let's only use `pathlib` instead. There's a table comparing the functions of both at the bottom of https://docs.python.org/3/library/pathlib.html

Reply:
Oh whoops! I saw all the formats, and remembered at some point the Python version was not high enough for f-strings, and assumed it meant it was not high enough for Pathlib. I am going to wait to check the version on marimba for the string formatting, and then use the appropriate things.

But, if there is already Pathlib, I am going to assume marimba has Python 3.6 and just use Pathlib. Nice catch1